lib/src/core/better_player_controller.dart

```diff
@@ -1159,6 +1159,14 @@ class BetterPlayerController {
     return videoPlayerController!.disablePictureInPicture();
   }
 
+  ///Disable Picture in Picture mode if it's enabled.
+  Future<void>? disablePictureInPictureManually() {
```

Comment:
I think, if it is called on Android device, always returns an error.
Is it correct?

If so, please add a comment 🙏

Reply:
> I think, if it is called on Android device, always returns an error.
> Is it correct?

Yes, it's correct. It will throw an exception if used on Android.

I added comments. Thank you.
https://github.com/dwango-nfc/betterplayer/pull/43/commits/bb49e8515e358c5bbf5e1ef0a2e9995088510880